pkg/point/request.go

```diff
@@ -1,26 +1,33 @@
 package point
 
-type CreateRequest struct {
+// Request allows the creation of a payment intention, that is, it represents a call that contains the details of a
+// transaction to be carried out and assigns it to a device.
+type Request struct {
```

Comment:
acho que assim fica melhor, o comentário atual acho que se encaixaria mais no método de criação
```suggestion
// Request is used to create a payment intention, sending the details of a transaction.
type Request struct {
```